packages/core/src/lib/notify.spec.ts

```diff
@@ -0,0 +1,83 @@
+import { describe, it, expect } from 'vitest';
+import { render, screen } from '@testing-library/svelte';
+import Notify from './notify.svelte';
+import NotifySlot from './notify.spec.svelte';
+
+describe('Notify', () => {
+  it('Renders notify element with appropriate title text, message text, if those attributes are specified', () => {
+    render(Notify, {
+      title: 'This is the title',
+      message: 'This is the message.',
+    });
+    expect(screen.getByText('This is the title')).toBeVisible();
+    expect(screen.getByText('This is the message.')).toBeVisible();
+  });
+});
+
+describe('Notify', () => {
```

Comment:
Could you please put all `it` statements in the same `describe`?

describe('Notify', () => {
  it('...', () => {...});
  it('...', () => {...});
  it('...', () => {...});
});